src/components/detailPanel/DetailPanel.js

```diff
@@ -1,19 +1,27 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
+
 import { NavLink } from 'react-router-dom';
 
-import { ReactComponent as CrossIcon } from 'assets/icons/cross.svg';
 import { ReactComponent as DocumentIcon } from 'assets/icons/document.svg';
 import DataTable from '../../shared/dataTable/DataTable';
 import SVGIcon from '../SVGIcon/SVGIcon';
+
 import { SpotTypes, StatusDisplayNames, SpotStatusTypes } from 'constants.js';
 import styles from './DetailPanel.module.scss';
 import classNames from 'classnames';
 import BlueLinkButton from 'shared/buttons/BlueLinkButton';
+
 import { spotTypeDisplayNames } from '../../constants';
+
 import { trackDownload } from 'helpers';
 import UserContext from '../../shared/user/UserContext';
 
+import { Heading, Button, Link } from '@datapunt/asc-ui';
+
+import { Close } from '@datapunt/asc-assets';
+import { HeaderStyle, ContentStyle } from './DetailPanelStyle';
```

Comment:
ik denk dat het beter is als je deze niet  in een andere file zet

Reply:
One of the reasons is that I want to skip the <ComponentName>Style.js from testing/coverrage. The other reason is that the number of style classes can be large and the overview of the component becomes difficult to read